doc/release/1.14.0-notes.rst

```diff
@@ -93,6 +93,13 @@ This change affects only ``float32`` and ``float16`` arrays.
 This is for pytest compatibility in the case of duplicate test file names in
 the different directories. As a result, ``run_module_suite`` no longer works,
 i.e., ``python <path-to-test-file>`` results in an error.
+``MaskedArray.squeeze`` never returns ``np.ma.masked``
```

Comment:
I'm now thinking that changing the `.gitattributes` for the changelog was a bad idea - it just creates silent rebase failures like these...